package-parser/tests/data/boundaries/api_data.json

```diff
@@ -42,20 +42,20 @@
                     "is_public": true,
                     "assigned_by": "POSITION_OR_NAME",
                     "docstring": {
-                        "type": "int in range (0, infinity)",
-                        "description": ""
+                        "type": "",
+                        "description": "int in range (0, infinity)"
                     }
                 },
                 {
-                    "name": "parameter_with_upper_bound",
+                    "name": "parameter_with_upper_bound_and_boundary",
                     "qname": "test.some_global_function.parameter_with_upper_bound",
                     "pname": "test/test/some_global_function/parameter_with_upper_bound",
                     "default_value": null,
                     "is_public": true,
                     "assigned_by": "POSITION_OR_NAME",
                     "docstring": {
-                        "type": "float in range (negative_infinity, 1)",
-                        "description": ""
+                        "type": "{'auto', 'person'} or float",
+                        "description": "float in range (negative_infinity, 1)"
```

Comment:
I don't understand the connection between type and description here.

Reply:
The documentation is like [PCA.iterated_power](https://scikit-learn.org/stable/modules/generated/sklearn.decomposition.PCA.html) and is meant to test, whether Types in UnionType are recognized at generating annotations. There are many examples in the documentation of sklearn, that have the type of the number in type and have the boundary in the description, if they have one. Most of the time it's only a number and a string, but it should work with EnumType and a number too.